oefenbestand-amber.txt

```diff
@@ -0,0 +1 @@
+Dit is een tekstbestand om te oefenen met github :)
```

Comment:
""suggestion
Dit is tekstbestand om een pro te worden in de wondere wereld van github